supercommand.go

```diff
@@ -71,17 +76,25 @@ type SuperCommandParams struct {
 	UserAliasesFilename string
 }
 
+// FlagAdder represents a value that has associated flags.
+type FlagAdder interface {
+	// AddsFlags adds the value's flags to the given flag set.
+	AddFlags(*gnuflag.FlagSet)
+}
+
 // NewSuperCommand creates and initializes a new `SuperCommand`, and returns
 // the fully initialized structure.
 func NewSuperCommand(params SuperCommandParams) *SuperCommand {
 	command := &SuperCommand{
-		Name:                params.Name,
-		Purpose:             params.Purpose,
-		Doc:                 params.Doc,
-		Log:                 params.Log,
+		Name:    params.Name,
+		Purpose: params.Purpose,
+		Doc:     params.Doc,
+		Log:     params.Log,
+		Aliases: params.Aliases,
```

Comment:
Any reason why this was moved?

Reply:
So that it fits with all the other exported fields.